modules/ngap_module/unit-tests/cmr_json_responses/README

```diff
@@ -0,0 +1,6 @@
+How to build these CMR granule_ur responses
+
+burl -o <granule_name>.json \
+'https://cmr.earthdata.nasa.gov/search/granules.umm_json_v1_4?pretty=true&collection_concept_id=<ccid>>&granule_ur=<gid>'
```

Comment:
Using the term `<gid>` is very confusing and should be changed. There is a granule "id" (aka "granule concept id", or as used in granule record, `concept-id`) using the granule concept id in place of `<gid>` will cause this request to fail. I suggest replacing `<gid>` with `<GranuleUR>` which is how the expected value is actually labeled in the CMR granule documents

Reply:
OK